src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -83,17 +83,29 @@ public class ExtraAnnotationViewInfo
       public IEnumerable<string> Nodes;
       public double FontSize;
       public string Background;
-
-      // TODO, QNTM-1099: Determine if these are required
-      // public string Id;
-      // public double Left;
-      // public double Top;
-      // public double Width;
-      // public double Height;
-      // public double InitialTop;
-      // public double InitialHeight;
-      // public double TextBlockHeight;
-  }
+      public string Id;
+
+        // TODO, QNTM-1099: Determine if these are required
+        // public string Id;
+        // public double Left;
+        // public double Top;
+        // public double Width;
+        // public double Height;
+        // public double InitialTop;
+        // public double InitialHeight;
+        // public double TextBlockHeight;
+
+        public override bool Equals(object obj)
+        {
+            var other = obj as ExtraAnnotationViewInfo;
+            return other != null &&
+                this.Id == other.Id &&
+                this.Title == other.Title &&
+                this.Nodes.SequenceEqual(other.Nodes) &&
+                this.FontSize == other.FontSize &&
+                this.Background == other.Background; 
+        }
```

Comment:
why the other parameters are not checked here?

Reply:
since they are not deserialized right now I am pretty sure they will be incorrect or slightly off, I'll add them so we get a baseline but I think we should avoid actually requiring them, if we can't recreate the annotation exactly, then thats fine - it's better than polluting the file with so much info. I could be totally off base. I'll change it and see.

Reply:
so I looked into this - as you said @ramramps these values do default or calculate somehow but they're not exactly the same as the values that we serialized last time. Not sure if thats a big deal or not, for now I'm not checking them in the tests so the tests pass, and I don't want to mess with deserialization of annotations in this PR. Lets deal with this investigation into what the consequence of letting these values calculate at runtime in another task.

For now I've added the checks and commented them out until we make a decision.

Reply:
I am sure those values will affect the group size / font / text placement.  if we want to remove that, then there is a way to calculate at run time (I never implemented it), based on  the text and font size.

Reply:
@Racel FYI need a decision / brief discussion on this.